### PR TITLE
Support assign set sentence to a variable.

### PR DESCRIPTION
**Pull request by CPWstatic**

### CODE REVIEW

**Comment by CPWstatic** [comment]
Jenkins go!

File: src/parser/parser.yy [hunk 1]
```diff
@@ -1169,6 +1169,9 @@ assignment_sentence
     : VARIABLE ASSIGN piped_sentence {
         $$ = new AssignmentSentence($1, $3);
     }
+    | VARIABLE ASSIGN set_sentence {
```
Comment (dangleptr) [inline src/parser/parser.yy]: piped_sentence is not one kind of set_sentence?

Comment (CPWstatic) [reply src/parser/parser.yy]: No, it isn't.

Comment (dangleptr) [reply src/parser/parser.yy]: IMO we should do that.  @dutor WDYT?

Comment (CPWstatic) [reply src/parser/parser.yy]: i'm ok with your opinion.

**Pull request merged**